### DISCUSSION TITLE
Raspberry Pi 5 and Electromage Output Expander

### DISCUSSION THREAD

**Discussion by reedsearle** [Pi4J/pi4j#401] [General]
I have recently been working with a Raspberry Pi 5 and LED strips to show Java student a visual demonstration of sorting methods.  I eventually stumbled on this site and bought the Output Extender from Electromage.  I was having trouble making the combination work until I noticed the following line in PixelblazeOutputExtender.java:
`        helper = new PixelBlazeOutputExpanderHelper("/dev/ttyS0");`
The Pi 5 does not have a /dev/ttyS0 as the UART conflicts from previous version have been resolved.  I modified the code to be:
`        helper = new PixelBlazeOutputExpanderHelper("/dev/ttyAMA0");`
Now, the neopixels work perfectly.  

Could a note be added to the Pixelblaze page to reflect this?

**Comment by taartspi** [comment]
thanks very much for the information.  I created  PullRequest with the updates.  Tom

**Comment by FDelporte** [comment]
Thanks @reedsearle and @taartspi for this info and the update of the website. I also added this info now in the jbang example code as a comment.
